documentation/index.md

```diff
@@ -0,0 +1,639 @@
+## Description
+
+The Maps App suite of applications is an example solution designed to showcase best practices in building cross platform applications around the ArcGIS Runtime SDK for .NET. The suite is built using Xamarin Forms and WPF and is comprised of four applications for four separate platforms with maximized code sharing.
```

Comment:
Can you rework this sentence to not read "example"? I think this will be an important part of our messaging going forward.

Reply:
Yep, good catch. See if you like my re-writing.